src/componentsNext/UserMenu/LogoutButton/EnterpriseLogout.tsx

```diff
@@ -0,0 +1,24 @@
+import { DropdownMenuItem } from "~/design/Dropdown";
```

Comment:
I wonder if it is necessary to have two different components for EnterpriseLogout and OpenSourceLogout.

It seems this is necessary because the authentication methods -- and the respective hooks -- are completely different in the two use cases and there is no other way to call these hooks conditionally. Am I correct? Then I think this is fine.

Reply:
The reason for that is, that `useAuth();` will throw an error in open source mode (the hook wants to read from a react context, but it does not have a provider) and we can not call a hook conditionally as React does not allow this.